store/errors.go

```diff
@@ -107,3 +110,67 @@ func (e InvalidAuthDataError) Error() string {
 	//      (empirically this checks out)
 	return strings.Join(es, "  ")
 }
+
+// InstallRefreshError conveys errors that were reported on otherwise overall successful install/refresh request.
+type InstallRefreshError struct {
```

Comment:
The term "InstallRefresh" feels a little bit awkward, in the sense that this is a pair of operations, yet it reads like an action ("install the refresh"). This PR is using it in quite a few places, and I think we're adding it just now mostly, so perhaps the right time to rethink. Can we come up with a more general term that would convey either operation?  Perhaps "SnapAction", which leverages the term already being adopted as part of these? So we'd have SnapActionError, store.SnapAction, Store.SnapAction(...), etc.

Reply:
used SnapAction